CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Bolt Next
+
+### Deprecations
+
+* **WARNING**: Support for plan method `Target.new(<uri>, <options>)` will be dropped in the near future. The plan method `Target.new(<options>)`, where `options` is a hash of target data and config, should be used instead.
```

Comment:
Ok, I think the best approach is to wait until #1522 is merged and do "Use `Target.new(<options>)`, where `options` is a hash of target data and config, instead. See [the docs](link-to-writing-plans-page) for more information and examples.